tox.ini

```diff
@@ -2,9 +2,14 @@
 envlist = py3
 
 [testenv]
+deps =
+    pytest
+    colorlog
```

Comment:
I think we need this dependency also in the `setup.cfg`.

I'm getting the following error locally:

```
ModuleNotFoundError: No module named 'colorlog'
```